src/Excel.php

```diff
@@ -115,7 +116,7 @@ public function queue($export, string $filePath, string $disk = null, string $wr
     {
         $writerType = FileTypeDetector::detectStrict($filePath, $writerType);
 
-        return $this->queuedWriter->store(
+        return QueueExportClass::dispatch(
```

Comment:
I just thought now we're queuing the export class itself instead of the export data and so I choosed this suffix. But yeah, `QueuedExport` sounds better. I will rename it.

Reply:
Ah okay got the reasoning now :) But yeah QueuedExport is better :)